instrumentation/que/lib/opentelemetry/instrumentation/que/version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Que
+      VERSION = '0.19.1'
```

Comment:
Since we started the 1.0 release candidates, we've decoupled instrumentation versions, so this can start at 0.1.0.

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/compare/4e646730ac5f7f3fde5ee479127a449d1e30d6ed..b8fff233a2309740f43a163a6a7330b78602e348